server/lib/resolvers/resolverMapper.js

```diff
@@ -23,9 +23,9 @@ module.exports = function (dataSources, resolverMappings, pubsub) {
       throw new Error('Missing request mapping for mapping: ' + resolverMappingName)
     }
 
-    if (_.isEmpty(resolverMapping.responseMapping)) {
-      throw new Error('Missing response mapping for mapping: ' + resolverMappingName)
-    }
+    // if (_.isEmpty(resolverMapping.responseMapping)) {
```

Comment:
The way I see it for 99% of the cases responseMapping will be empty (it will be always promise) so this makes me think that after merging this PR we may really review this implementation.